.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: ci
+on: [push]
+jobs:
+  ubuntu:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        elixir-version: ['1.9', '1.10']
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-elixir@v1
+        with:
+          otp-version: '22.0'
```

Comment:
It may be worth running checks for OTP 23 as well as 22, which we can add to the matrix, again no major time differences, and also something Craig will have thoughts on. (This may be me overcomplicating things!)